app/assets/stylesheets/_summary-card.scss

```diff
@@ -0,0 +1,29 @@
+.summary-card {
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  @include box-shadow(0 1px 4px rgba(0,0,0,.3));
+
+  .sc-row {
```

Comment:
As we start building out more of our UI, should we start using BEM methodology to name things? Just to start a proper pattern here. Reference: http://getbem.com/naming/

So in this case, this could be changed to be: `&__row {`, which would out put as `.summary-card__row`.

It might might end up being verbose but I think it'll set us up for specificity with one or two classes, as well as make it easier to search for elements in the code base.

Open to discussion, though. What do you think @T-Dnzt?